frontend/src/concepts/modelRegistry/apiHooks/useModelArtifactsByVersionId.ts

```diff
@@ -8,15 +8,19 @@ import { ModelArtifactList } from '~/concepts/modelRegistry/types';
 import { useModelRegistryAPI } from '~/concepts/modelRegistry/context/ModelRegistryContext';
 
 const useModelArtifactsByVersionId = (modelVersionId?: string): FetchState<ModelArtifactList> => {
-  const { api } = useModelRegistryAPI();
+  const { api, apiAvailable } = useModelRegistryAPI();
   const callback = React.useCallback<FetchStateCallbackPromise<ModelArtifactList>>(
     (opts) => {
       if (!modelVersionId) {
         return Promise.reject(new NotReadyError('No model registeredModel id'));
       }
+      if (!apiAvailable) {
```

Comment:
shouldn't this precede `if (!modelVersionId)` - I think one can't get `modelVersionId` when the API isn't available?